src/com/cloudogu/ces/cesbuildlib/Git.groovy

```diff
@@ -143,6 +143,26 @@ class Git implements Serializable {
         return !getTag().isEmpty()
     }
 
+    /**
+     * @return true if the specified tag exists
+     */
+    boolean tagExists(String tag){
+        if (credentials) {
```

Comment:
Couldn't you extend `executeGitWithCredentials()` to return stdout and reuse it?

Reply:
Unfortunately the `executeGitWithCredentials` method was changed to use the status code of the git command. 
It seems not to be possible to read status code and stdout at the same time.
I found a workarount but i'm not sure if this is a good practice. I save the command output to a file and read it after executing while i'm getting the status code on the default way.
Do you think this is ok or do we need another way? Do you have any ideas?